scrapy/pipelines/media.py

```diff
@@ -16,6 +19,7 @@
 class MediaPipeline(object):
 
     LOG_FAILED_RESULTS = True
+    ALLOW_REDIRECTS = False
```

Comment:
I'm fairly new to detailed review/edits/commits on github.  I've done full pullrequests before.  IS there a way for me to pick up and make the changes you suggest and submit them under this pullrequest as an edit?  Or must the original contributor do it?